docker-compose.yaml

```diff
@@ -5,13 +5,14 @@ services:
     build: .
     #image: ngoduykhanh/wireguard-ui:latest
     container_name: wgui
+    cap_add:
+      - NET_ADMIN
+    network_mode: host
     environment:
       - SENDGRID_API_KEY
       - EMAIL_FROM
       - EMAIL_FROM_NAME
       - SESSION_SECRET
-    ports:
-      - 5000:5000
```

Comment:
Why don't we want to expose the port here?

Reply:
Hi @ngoduykhanh , thanks for the review. 

In order to get the status from the `wg` command the docker-compose runs with network-mode host. 
This means that we do not need to expose any port